projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -188,6 +188,8 @@ const Article = ({
                 path={path}
                 scrollEnabled={true}
                 useWebKit={false}
+                allowsInlineMediaPlayback={true} // need this along with `mediaPlaybackRequiresUserAction = false` to ensure videos in twitter embeds play on iOS
```

Comment:
Just a check here that this doesnt affect the youtube videos?

Reply:
Particularly tablet

Reply:
Yeah, this won't affect youtube videos playing full screen on mobile. 
I've checked current behavior on iPad which plays video inline and this change doesn't affect that.